scripts/setup-docker.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+export BROKER_URL=amqp://olympia:olympia@rabbitmq/olympia
+export DATABASE_URL=mysql://root:@mysqld/olympia
+export ELASTICSEARCH_LOCATION=elasticsearch:9200
+export MEMCACHE_LOCATION=memcached:11211
+export MYSQL_DATABASE=olympia
+export MYSQL_ROOT_PASSWORD=docker
+export OLYMPIA_SITE_URL
+export PYTHONDONTWRITEBYTECODE=1
+export PYTHONUNBUFFERED=1
+export RECURSION_LIMIT=10000
+export REDIS_LOCATION=redis://redis:6379/0?socket_timeout=0.5
+export TERM=xterm-256color
+
+# initialize_db:
+python manage.py reset_db --noinput
+python manage.py syncdb --noinput
+python manage.py loaddata initial.json
+python manage.py import_prod_versions
+schematic --fake src/olympia/migrations/
+#python manage.py createsuperuser
+#python manage.py loaddata zadmin/users
+
+# update_assets:
+make update_assets
+
+#populate_data:
+export NUM_ADDONS=10
+export NUM_THEMES=$NUM_ADDONS
+python manage.py reindex --wipe --force --noinput
```

Comment:
To avoid https://github.com/mozilla/addons-server/issues/2706 I've copied this from the `Makefile` and call `reindex` before the `generate_` tasks.